File: user-service/src/main/java/rs/raf/user_service/repository/ActuaryLimitRepository.java
package rs.raf.user_service.repository;

import org.springframework.data.jpa.repository.JpaRepository;
import rs.raf.user_service.domain.entity.ActuaryLimit;

import java.util.Optional;

public interface ActuaryLimitRepository extends JpaRepository<ActuaryLimit, Long> {
    Optional<ActuaryLimit> findByEmployeeId(Long employeeId);

}

File: user-service-kotlin/src/main/kotlin/pack/userservicekotlin/repository/ActuaryLimitRepository.kt
package pack.userservicekotlin.repository

import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository
import pack.userservicekotlin.domain.entities.ActuaryLimit
import java.util.*

@Repository
interface ActuaryLimitRepository : JpaRepository<ActuaryLimit, Long> {
    fun findByEmployeeId(employeeId: Long): Optional<ActuaryLimit>
}
